wireless_modules/mqtt_test/boot.py

```diff
@@ -0,0 +1,27 @@
+from config import ESSID
+from config import PASSWORD
+
+def connect(eesid, password):
+    import network
+    sta_if = network.WLAN(network.STA_IF)
```

Comment:
I had to google what `STA_IF` meant. Let's rename `sta_if` to `station_interface` or `network_interface` instead? Thoughts?

Reply:
I think it is a good idea! `sta_if` is confusing when you first read it and `station_interface` is much more clear. I like it!